sizes/output.go

```diff
@@ -489,6 +506,20 @@ func (s *HistorySize) contents(refGroups []RefGroup) tableContents {
 		rgis = append(rgis, rgi.Indented(indent))
 	}
 
+	var refCountSection section
+	if !opts.WithoutReferenceCount {
+		refCountSection = S(
+			"References",
+			I("referenceCount", "Count",
+				"The total number of references",
+				nil, s.ReferenceCount, metric, "", 25e3),
+			S(
+				"",
+				rgis...,
+			),
+		)
+	}
+
```

Comment:
This approach seems to depend on the fact that even though the uninitialized `section` is inserted into the table, it doesn't output anything. I think that's correct :heavy_check_mark: